compassbearing.py

```diff
@@ -0,0 +1,40 @@
+# LICENSE: public domain
+import math

```

Comment:
Ah I see this is already coming from the examples repo [here](https://github.com/brooksandrew/postman_problems_examples/blob/master/compass_bearing.py).